mitiq/calibration/tests/test_calibration.py

```diff
@@ -84,8 +99,26 @@ def execute(circuit, noise_level=0.001):
 )
 
 
+def execute(circuit, noise_level=0.001):
+    circuit = circuit.with_noise(cirq.amplitude_damp(noise_level))
+
+    result = cirq.DensityMatrixSimulator().run(circuit, repetitions=100)
+    bitstrings = np.column_stack(list(result.measurements.values()))
+    return MeasurementResult(bitstrings)
+
+
+def non_cirq_execute(circuit):
+    # Ensure test circuits are converted to user's frontend by the Calibrator
+    assert not isinstance(circuit, cirq.Circuit)
+    circuit, circuit_type = convert_to_mitiq(circuit)
+    # Pennylane and Braket conversions discard measurements so we re-append
+    if circuit_type in ["braket", "pennylane"]:
+        circuit.append(cirq.measure(q) for q in circuit.all_qubits())
+    return execute(circuit)
```

Comment:
Ha! Really interesting executor to test across all frontend types. Very nifty.

Do we know why pennylane and braket discard measurements in the conversion? IIRC braket doesn't have measurement gates, but I'm unsure about pennylane.